spec/models/direct_file_data_spec.rb

```diff
@@ -466,22 +466,6 @@
       end
     end
 
-    context "when there are dependents in AZ, the months_in_home is not populated" do
-      let(:xml) { StateFile::DirectFileApiResponseSampleService.new.read_xml('az_johnny_mfj_8_deps') }
-
-      it 'sets the months_in_home to nil' do
-        expect(described_class.new(xml).dependents).to be_all { |d| d.months_in_home.nil? }
-      end
-    end
-
-    context "when there are dependents in NY, the months_in_home IS populated" do
-      let(:xml) { StateFile::DirectFileApiResponseSampleService.new.read_xml('ny_matthew') }
-
-      it 'sets the months_in_home' do
-        expect(described_class.new(xml).dependents).to be_all { |d| d.months_in_home.present? }
-      end
-    end
```

Comment:
We updated our `months_in_home` logic to use the DF JSON, so these tests are obsolete now.

Reply:
Confirmed